ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.17__multi-nightly.yaml

```
@@ -1861,22 +1861,21 @@ tests:
     test:
     - chain: openshift-e2e-test-qe
     workflow: cucushift-installer-rehearse-gcp-ipi-sno
-- as: gcp-ipi-labels-tags-filestore-csi-fips-tp-amd-f28-destructive
-  cron: 32 16 7 * *
+- as: gcp-ipi-labels-tags-filestore-csi-fips-amd-f28-destructive
+  cron: 58 15 27 * *
   steps:
     cluster_profile: gcp-qe
     dependencies:
       OPENSHIFT_INSTALL_RELEASE_IMAGE_OVERRIDE: release:multi-latest
     dependency_overrides:
       OO_INDEX: quay.io/openshift-qe-optional-operators/aosqe-index:v4.17
     env:
-      FEATURE_SET: TechPreviewNoUpgrade
       FIPS_ENABLED: "true"
     test:
     - chain: openshift-e2e-test-qe-destructive
     workflow: cucushift-installer-rehearse-gcp-ipi-user-labels-tags-filestore-csi
-- as: gcp-ipi-user-labels-tags-filestore-csi-tp-arm-f14
-  cron: 13 9 10,26 * *
+- as: gcp-ipi-user-labels-tags-filestore-csi-arm-f14
```

```suggestion
- as: gcp-ipi-labels-tags-filestore-csi-arm-f14
```

To keep consistent name with the destructive job name.